internal/ingress/metric/main_test.go

```diff
@@ -0,0 +1 @@
+package metric
```

Comment:
what's this used for?

Reply:
makes coveralls count the package in our overall test coverage